### PR TITLE
pci_hotplug_check:Add aarch64 bus designation

### PR DESCRIPTION
**Pull request by zhenyzha**
Add aarch64 bus designation

ID: 2033170
Signed-off-by: Zhenyu Zhang <zhenyzha@redhat.com>

### CODE REVIEW

**Comment by zhenyzha** [comment]
(1/1) Host_RHEL.m9.u0.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.9.0.0.aarch64.io-github-autotest-qemu.block_hotplug_check.arm64-pci: PASS (674.43 s)

@PaulYuuu Could you pls help review when you are free. thanks.

File: qemu/tests/cfg/block_hotplug_check.cfg [hunk 1]
```diff
@@ -17,7 +17,7 @@
     force_create_image = yes
     force_create_image_image1 = no
     soundcards = ""
-    q35:
+    q35, arm64-pci, arm64-mmio:
```
Comment (PaulYuuu) [inline qemu/tests/cfg/block_hotplug_check.cfg:20]: Seems arm64-mmio does not use PCIe bridge.

Comment (zhenyzha) [reply qemu/tests/cfg/block_hotplug_check.cfg]: Thanks for the reminder, it has been corrected.

**Comment by zhenyzha** [comment]
(1/1) Host_RHEL.m9.u0.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.9.0.0.aarch64.io-github-autotest-qemu.block_hotplug_check.arm64-pci: PASS (683.26 s)

@PaulYuuu Could you pls help review when you are free. thanks.

**Review by PaulYuuu — APPROVED** [review]

**Pull request merged**